### PR TITLE
[JENKINS-67847] booleanRadio button is not shown correctly

### PR DESCRIPTION
**Pull request by benebsiny**
See [JENKINS-67847](https://issues.jenkins-ci.org/browse/JENKINS-67847).  An issue in Jenkins 2.335 but not an issue in Jenkins 2.332.

radioBlock is implemented in booleanRadio and it seems well.
Boolean Radio

### Proposed changelog entries

* Place text correctly on boolean radio controls (regression in 2.335).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- ~~If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))~~
- ~~If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).~~

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]
I built this change and tested it locally to confirm that the text placement of the "Failed" label on the radio button from the quality gate section of the warnings ng plugin is not rendered over the top of the radio button.  Thanks very much!

image

**Review by timja — APPROVED** [review]
Untested

**Comment by timja** [comment]
@MarkEWaite did you check data saved correctly? Or just the rendering?

**Review by NotMyFault — APPROVED** [review]
Tested it additionally with other buttons from other plugins and core itself, works great so far 👍🏻

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by basil** [comment]
Before this PR, `jenkins.plugins.http_request.HttpRequestStepRoundTripTest` consistently passes. After this PR, `jenkins.plugins.http_request.HttpRequestStepRoundTripTest` consistently fails.

**Comment by daniel-beck** [comment]
Seems to have been an intentional change per https://github.com/jenkinsci/jenkins/pull/5923#issuecomment-1022201470 where I mentioned the same issue for checkboxes?

**Comment by timja** [comment]
> Seems to have been an intentional change per https://github.com/jenkinsci/jenkins/pull/5923#issuecomment-1008156880

Can't tell what you are linking to, GitHub doesn't link well on PRs with lots of comments or resolved / unresolved conversations. Could you quote it instead?

(this is on my list to look at unless anyone beats me to it)

**Comment by daniel-beck** [comment]
> Can't tell what you are linking to, GitHub doesn't link well on PRs with lots of comments or resolved / unresolved conversations. Could you quote it instead?

Works for me, also snippets are shown in the tooltip as well. Also, your resolutions of all my comments makes this extra super painful, please stop doing that.

Your response to me pointing out this problem was:

> There’s a PR for the config file provider to fix up its UI https://github.com/jenkinsci/config-file-provider-plugin/pull/160

**Comment by timja** [comment]
https://github.com/jenkinsci/jenkins/pull/6311

**Pull request merged**
